packages/utils/src/ts-eslint/RuleTester.ts

```diff
@@ -169,23 +169,26 @@ declare class RuleTesterBase {
    * @param text a string describing the rule
```

Comment:
Existing code, not a blocker: this JSDoc was already out of date. I have a pending issue on my own personal template repo for finding a good set of lint rules that make sure JSDocs/TSDocs don't get out of sync: https://github.com/JoshuaKGoldberg/template-typescript-node-package/issues/17#issuecomment-1286232744. I can file an issue here to add it in once I do.

Reply:
*technically* the jsdoc is correct, it just applies to the `RuleTesterTestFrameworkFunction` type, not the property itself.

so should defs move the jsdoc onto the type itself